dakara_server/library/base_test.py

```diff
@@ -141,6 +149,20 @@ def check_work_json(self, json, expected_work):
         self.assertEqual(json['title'], expected_work.title)
         self.assertEqual(json['subtitle'], expected_work.subtitle)
         self.check_work_type_json(json['work_type'], expected_work.work_type)
+        # alternative titles
+        expected_alt_titles = expected_work.alternative_titles.all()
+        self.assertEqual(len(json['alternative_titles']),
+                         len(expected_alt_titles))
+        for alt_title, expected_alt_title in zip(json['alternative_titles'],
+                                                 expected_alt_titles):
+            self.check_work_alternative_title_json(
+                alt_title, expected_alt_title)
+
+    def check_work_alternative_title_json(self, json, expected_work_alt_title):
```

Comment:
Why use a separated function?

Reply:
It seems to me that the code would more intelligible that way.

Reply:
The function is a one-liner and is used only once.

Reply:
Put that way, it is true it is quite unnecessary.